frontend/class-twitter.php

```diff
@@ -460,7 +460,10 @@ private function image_thumbnail_output() {
 	 * @return bool
 	 */
 	private function image_from_content_output() {
-		if ( preg_match_all( '`<img [^>]+>`', $GLOBALS['post']->post_content, $matches ) ) {
+		global $post;
+		$content = apply_filters( 'wpseo_pre_analysis_post_content', $post->post_content, $post );
```

Comment:
This filter needs documentation, and I don't see a good reason for `global $post` when you could just copy the `$GLOBALS` statement.